src/platforms/dotnet/guides/azure-functions-worker/config.yml

```
@@ -0,0 +1,2 @@
+title: Azure Functions Isolated Worker
```

Let's leave "Isolated Worker" out of the title, so it fits in the side bar.  We can qualify it in the doc text.  If we ever later add a page for a different model, we can decide how to adjust at that time.

```suggestion
title: Azure Functions
```